common/constants.go

```diff
@@ -308,3 +308,14 @@ const (
 	ConfigStemDevice   = ConfigStemAll
 	ConfigStemSecurity = ConfigStemAll
 )
+
```

Comment:
add a comment to what these constants are
```suggestion
// Constants related to command/query request topic
```

Reply:
Adde a comment